src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -29,6 +31,8 @@
     private Encoder rightEncoder;
     private PIDGyro gyro;
     private DifferentialDrive drive;
+    NetworkTableInstance inst; // = NetworkTableInstance.getDefault();
+    NetworkTable table; // inst.getTable("Vision Correction Table");
```

Comment:
Stop 👏 leaving 👏 commented 👏 out 👏 code 👏👏👏👏👏👏👏👏👏

Reply:
SadCat... but fixed it